README.md

```diff
@@ -363,6 +364,27 @@ signIn.off('pageRendered');
 signIn.off('pageRendered', onPageRendered);
 ```
 
+## getTransaction(stateToken)
+
+Returns [authentication transaction information](https://developer.okta.com/docs/api/resources/authn#transaction-model) given a `stateToken`. If a `stateToken` is not provided, it will fetch the current token from a saved browser cookie.
```

Comment:
Maybe I remember wrong, but from the conversations didn't you guys decided to go for 2 different functions? one that use the stateToken as input and one that use the one from the cookie?

Reply:
We decided that having two methods didn’t make sense, as relying on the browser cookie isn’t a feasible use case at this stage. However, the `resume` method queries for the token if one is not provided, so we made sure to document that here.

Further, the only viable use case where you’d want to rely on the stateToken cookie is after primary authentication (in the middle of a Widget view). We are going to encourage devs to listen to change events if they need it mid-auth.